packages/grafana-ui/src/components/Button/ToolbarButton.tsx

```diff
@@ -84,6 +87,7 @@ export const ToolbarButton = forwardRef<HTMLButtonElement, Props>(
         {children && !iconOnly && <div className={contentStyles}>{children}</div>}
         {isOpen === false && <Icon name="angle-down" />}
         {isOpen === true && <Icon name="angle-up" />}
+        {isHighlighted === true && <div className={styles.highlight} />}
```

Comment:
```suggestion
        {isHighlighted && <div className={styles.highlight} />}
```
`isHighlighted` can only be a boolean or undefined so this check can be simplified.